spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RecordFilterStrategy.java

```diff
@@ -58,4 +58,8 @@ default List<ConsumerRecord<K, V>> filterBatch(List<ConsumerRecord<K, V>> record
 		return records;
 	}
 
+	default boolean ignoreEmptyBatch() {
```

Comment:
For any new public API we need Javadoc, plus `@since`

Reply:
Thanks for your comment 🙇‍♂️ 
I added java docs for `ignoreEmptyBatch()`!